test/app-forms.spec.js

```diff
@@ -308,6 +312,22 @@ describe('forms that have caused bugs', () => {
     expect(result.errors).to.be.empty;
   });
 
+  it('#234 - fill datetime field', async () => {
+    const result = await harness.fillForm('bug_234', ['2023-01-01 1:22']);
+    expect(result.errors).to.be.empty;
+  });
+
+  it('#234 - fill datetime expected input format', () => {
+    return expect(harness.fillForm('bug_234', ['2023-01-01']))
+      .to.be.rejectedWith('expect input in format');
+  });
+
+  it('#234 - datetime field is required', async () => {
+    const result = await harness.fillForm('bug_234', ['2023-01-01 x']);
```

Comment:
Why not fail with the input format error?

Reply:
Currently the form filler is pretty dumb: it just checks that there is a ` ` delimited input and it puts the first value in the first box + the second input in the second box. This test has a valid input (delimited string) but it fails Enketo's validations.

I think this is probably a good design decision for a test tool. Ideally, test tools would be able to test whatever inputs a human can provide.  If the form filler only filled vaild well-formed inputs in the proper format, then it is actually a less useful test tool.

Reply:
oh i see, thanks for clearing that up